.chloggen/export-logs-resource-info-based-configuration.yaml

```diff
@@ -0,0 +1,20 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: datasetexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Allow include Logs resource info export to DataSet based on new export_resource_info_on_event configuration. Fix timestamp handling."
```

Comment:
This is a left over from another PR from @zdaratom which for some reason was missing from his PR.

Reply:
not sure I follow, what is this changelog entry doing here?

Reply:
We missed / forgot to add changelog as part of that PR (https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/23250), so I added it here.

Reply:
Please create a chore pr to add the changelog asap

Reply:
I see the changelog entry there: .chloggen/export-logs-resource-info-based-configuration.yaml so I don’t think it was missed.

Reply:
@atoulme I'm not sure I understand your comment. I don't see that changelog file in the main branch (https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/.chloggen), I only see it in this one where I added it.

Am I missing something? Thanks.

Reply:
I went ahead and removed that changelo file.